web/index.html

```diff
@@ -858,7 +857,7 @@ <h2 id="discussion">What have we learned?</h2>
     <h2>Future considerations</h2>
     <p>
       Although MCTS' good performance on HumanEval didn't carry through to the VerilogEval dataset,
-      we believe there is opportunity to make it more robust to fluctuations in the underlying LLM
+      we believe there is an opportunity to make it more robust to fluctuations in the underlying LLM
```

Comment:
don't agree with this fix, but could change to "... we believe there are opportunities ..." if you'd prefer?